kirin/utils.py

```diff
@@ -286,7 +286,7 @@ def manage_db_error(
 
 
 def manage_db_no_new(connector_type: str, contributor_id: str) -> None:
```

Comment:
Same: let's clean it all (also in callers)
```suggestion
def manage_db_no_new(contributor_id: str) -> None:
```

Reply:
Done, see https://github.com/CanalTP/kirin/pull/426#issuecomment-985438533.